modules/site-navigation/rest-fields/page-user-can.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Elementor\Modules\SiteNavigation\Rest_Fields;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Page_User_Can {
```

Comment:
why do you need a different endpoint for user can?

Reply:
Its not an endpoint.
I'm adding a field to the 'pages' resource.
This field will be enriched when we request `/wp/v2/pages/?_fields=user_can`